tests/rex-uptime/test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Running remote execution test: uptime"
+
+job_template='Run Command - Script Default'
+SMART_PROXY_NAME=$(hostname -f)
+ORG="Test Org"
+REX_ORG="Remote Execution Org $$"
+
+OLD_ORGS=$(hammer --output csv --no-headers --show-ids proxy info --name "${SMART_PROXY_NAME}" --fields Organizations)
+
+hammer organization create --name "${REX_ORG}"
+hammer host update --name "${SMART_PROXY_NAME}" --new-organization "${REX_ORG}"
+hammer proxy update --name "${SMART_PROXY_NAME}" --organizations "${OLD_ORGS},${REX_ORG}"
+
+hammer job-invocation create \
```

Comment:
This command blocks until the job finishes. If the job failed, the exit code is non-zero so we don't really have to invoke the following two hammer commands and parse them for details.

Reply:
I tried simplify the logic and drop the additional calls.